MyFirstContribution/script.js

```diff
@@ -1,85 +1,95 @@
-    //This data contains the data the is being dipslayed as card in the website
-    // Add your details here
-    const data = [
-        {
-            "name":"Jitender Singh Chhapola",
-            "occupation": "Full Stack Software Developer",
-            "username": "niteshjitender",
-            "github": "https://github.com/niteshjitender",
-            "location": "India",
-            "favouriteQuote":"Improving, Learning, Enjoying.",
-            "profilePic":"./userImageData/niteshjitender.jpeg"
-        }
-        //Add an object here containing your data
-    ]
-    
+//This data contains the data the is being dipslayed as card in the website
+// Add your details here
+const data = [
+    {
+        "name": "Jitender Singh Chhapola",
+        "occupation": "Full Stack Software Developer",
+        "username": "niteshjitender",
+        "github": "https://github.com/niteshjitender",
+        "location": "India",
+        "favouriteQuote": "Improving, Learning, Enjoying.",
+        "profilePic": "./userImageData/niteshjitender.jpeg"
+    },
+    {
+        "name": "Sayam Alvi",
+        "occupation": "React Developer",
+        "username": "sayamalvi",
+        "github": "https://github.com/sayamalvi",
+        "location": "India",
+        "favouriteQuote": "There are no shortcuts to success. You gotta hustle hard.",
+        "profilePic": "./userImageData/sayamalvi.jpeg"
```

Comment:
Profile pic is not shown in your card.
Please change the extension of the profile pic to as same as in the file name.